### DISCUSSION TITLE
Cannot find module 'inline-style-prefixer/lib/plugins/backgroundClip'

### DISCUSSION THREAD

**Discussion by captDaylight** [necolas/react-native-web#2429] [Help]
I'm getting this error all of a sudden (Tamagui inherits from `react-native-web`):

```
Tamagui Webpack Loader Error:
   Cannot find module 'inline-style-prefixer/lib/plugins/backgroundClip'
```

When I look at `inline-style-prefixer` I see there have been some very recent changes that remove `backgroundClip`. https://github.com/robinweser/inline-style-prefixer/blame/master/CHANGELOG.md

I see that `react-native-web` has `"inline-style-prefixer": "^6.0.0",`

Is anyone else experiencing this?

**Comment by necolas** [comment]
> ### 6.0.4 - hotfix bringing back a no-op `backgroundClip` plugin to preserve clients using a custom configuration

**Comment by captDaylight** [comment]
Since posting this, I'm seeing the recent PR https://github.com/necolas/react-native-web/pull/2427
